controllers/datadogagent/component/clusteragent/default.go

```diff
@@ -4,15 +4,28 @@ import (
 	"fmt"
 	"strconv"
 
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	apicommon "github.com/DataDog/datadog-operator/apis/datadoghq/common"
 	apicommonv1 "github.com/DataDog/datadog-operator/apis/datadoghq/common/v1"
 	apiutils "github.com/DataDog/datadog-operator/apis/utils"
 	"github.com/DataDog/datadog-operator/controllers/datadogagent/component"
+	"github.com/DataDog/datadog-operator/pkg/defaulting"
 )
 
+// NewDefaultClusterAgentDeployment return a new default cluster-agent deployment
+func NewDefaultClusterAgentDeployment(dda metav1.Object) *appsv1.Deployment {
+	deployment := component.NewDeployment(dda, apicommon.DefaultClusterAgentResourceSuffix, component.GetClusterAgentName(dda), component.GetClusterAgentVersion(dda), nil)
+
+	podTemplate := NewDefaultClusterAgentPodTemplateSpec(dda)
+	deployment.Spec.Template = *podTemplate
+	deployment.Spec.Replicas = apiutils.NewInt32Pointer(1)
```

Comment:
maybe should put in a constant

Reply:
agree

Reply:
done in https://github.com/DataDog/datadog-operator/pull/491/commits/c61514c7b095598bd22bc5959a00946c3df99f82